### ISSUE TITLE
Tests are skipped when running on Jenkins

### ISSUE THREAD

**Issue by nthState** [sketch-hq/SketchAPI#682]
Lots of the SketchAPI tests are skipped in Sketch's Jenkins CI env. Can we fix this? Having tests that only run when a developer happens to run the tests locally against whatever arbitrary version of Sketch they have installed is not all that helpful.

Test using: `npm run run-tests:no-variant`

If you sync the SketchAPI and run the unit tests, four fail. 

- AssetLibrary failure, it happens because the Native code (MSAssetLibraryType) was updated and the
JS wasn't

```
const LibraryTypeMap = {
  0: 'Internal',
  1: 'User', // Is this a library added by a user.
  2: 'Remote', // Is this a library that can be updated using an appcast.
}
```

We can fix this, but we may introduce a breaking change:

```
const LibraryTypeMap = {
  0: 'Internal',
  1: 'LocalUser', // Previously `User`, is this a library added by a user.
  2: 'RemoteUser', // Previously `Remote`, is this a library that can be updated using an appcast.
  3: 'RemoteTeam', // Is this a library hosted on Sketch for Teams
  4: 'RemoteThirdParty',
}
```

- Curve point selection tests "CurvePoint › should show if a point is selected"

The method ``` + (NSIndexPath *)indexPathForPoint:(NSUInteger)point ofShape:(NSUInteger)shape;```
has been removed.

MSVectorComponentSpecifier needs: @objc(MSVectorComponentSpecifier) and the test may change to: 

```
const path = NSIndexPath.indexPathForItem_inSection(0, 0)

    const component = MSVectorComponentSpecifier.alloc().initWithComponent_indexPath(
      0,
      path
    )

    eventHandler.pathController().setSelectionComponent(component)
    console.log(shape.points[0].isSelected())

    expect(shape.points[0].isSelected()).toBe(true)
```

However, changing this test caused another crash which needs investigating

- "Library › should be able to get the document"

TODO

- "Library › should be able to get the list of symbols to be imported"

TODO

Example place in code where tests are skipped:
https://github.com/sketch-hq/SketchAPI/blob/develop/Source/dom/models/__tests__/Document.test.js#L180

**Comment by mathieudutour** [comment]
An easy fix is to just add `sketch-hq/SketchAPI` to the `plugin-tests` repo (or something like this, can't remember how it's called). It will run all the tests daily with the latest develop IIRC.

**Comment by christianklotz** [comment]
Fixed with #743
